app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CustomersController < ApplicationController
+  def index
+    customers = Customer.all
+
+    render(
+      json: customers.as_json(only: [:id, :name, :registered_at, :address, :city, :state, :postal_code, :phone, :account_credit, :movies_checked_out_count]),
+      status: :ok
+    )
+  end
+
+  def show
+    customer = Customer.find_by(id: params[:id])
+    if customer
+      render(
+        json: customer.as_json(only: [:name, :registered_at, :address, :city, :state, :postal_code, :phone, :account_credit, :movies_checked_out_count]),
+        status: :ok
+      )
+    else
+      render json: {nothing: true}, status: :not_found
+    end
+  end
+
+  private
+
+  def customer_params
+    params.require(:customer).permit(:id, :name, :registered_at, :address, :city, :state, :postal_code, :phone, :account_credit, :movies_checked_out_count)
+  end
```

Comment:
You shouldn't need to define strong params if you don't have a `create` or `update` action. As far as I can tell this method is never called.